tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -6065,6 +6069,10 @@ def get_matmul(arr):
         pytest.param("c", marks = [
             pytest.mark.skip(reason="Needs a C printer see https://github.com/pyccel/pyccel/issues/791"),
             pytest.mark.c]
+        ),
+        pytest.param("python", marks = [
+            pytest.mark.python,
+            pytest.mark.skip(reason="Missing Python printer")]
```

Comment:
Would you mind adding the python printers? For most classes adding the python printer is a 2 line fix so it's worth doing straight away

Reply:
sure